tools/IoTEdgeDevOps/VstsPipelineSync/VstsBuildBatchUpdate.cs

```diff
@@ -15,14 +15,17 @@ public class VstsBuildBatchUpdate
         readonly DevOpsAccessSetting devOpsAccessSetting;
         readonly string dbConnectionString;
         Dictionary<string, Dictionary<BuildDefinitionId, DateTime>> lastUpdatePerBranchPerDefinition;
+        HashSet<string> branches;
 
-        public VstsBuildBatchUpdate(DevOpsAccessSetting devOpsAccessSetting, string dbConnectionString)
+        public VstsBuildBatchUpdate(DevOpsAccessSetting devOpsAccessSetting, string dbConnectionString, HashSet<string> branches)
         {
             ValidationUtil.ThrowIfNull(devOpsAccessSetting, nameof(devOpsAccessSetting));
```

Comment:
Why can't we use the Preconditions from Edge.Util?

Reply:
I just don't want to have dependency from edge.util.  If someone changed anything in edge.util, they don't know that may break anything in devops solution.